jbpm-installer/src/main/resources/db/ddl-scripts/sybase/quartz_tables_sybase.sql

```diff
@@ -114,8 +25,8 @@ go
 
 create table QRTZ_CRON_TRIGGERS (
 SCHED_NAME varchar(120) not null,
-TRIGGER_NAME varchar(80) not null,
-TRIGGER_GROUP varchar(80) not null,
+TRIGGER_NAME varchar(200) not null,
+TRIGGER_GROUP varchar(200) not null,
```

Comment:
One thing I realized now - does it have to be 200? Can't we just leave it?

Reply:
This wasn't updated with official quartz tables
https://github.com/quartz-scheduler/quartz/blob/d42fb7770f287afbf91f6629d90e7698761ad7d8/quartz-core/src/main/resources/org/quartz/impl/jdbcjobstore/tables_sybase.sql#L117-L118
As these columns are part of the foreign_key referenced by QRTZ_TRIGGERS (SCHED_NAME,TRIGGER_NAME,TRIGGER_GROUP), they couldn't have lower size.